bin/promote-nightly-to-release

```diff
@@ -61,7 +61,7 @@ git commit DEBIAN_REPOSITORIES -m "Adding DISTRO-${MAJOR}.${MINOR} apt repositor
 statusline "Pushing packaging branch..."
 git push -u origin "HEAD:HHVM-${VERSION}"
 
-SCRATCHDIR="$(mktemp -d)"
+SCRATCHDIR="$(mktemp -dt hhvm.XXXXXXXX)"
```

Comment:
slight gnu'ism here

-t is 'prefix' on BSD (including macos); this is fine, just results in stuff like `/var/folders/m0/zhq4ljv51s54qjf274hw_0pc0000gn/T/hhvm.XXXXXXXX.fnMdULrg`

Reply:
That's probably fine, the extra Xs on MacOS shouldn't break anything.

This is just so I can do `rm -rf /tmp/hhvm*` so that a worker doesn't run out of disk space if it processes multiple tasks.

Reply:
yeah, works fine :)

just checking: there's not something assuming it ends up in /tmp/ hiding here , right ? :)

Reply:
There is the `rm -rf /tmp/hhvm*` in `worker.sh` which assumes that, but the worker code is only meant to run on AWS Ubuntu, there's lots of other unportable stuff there.

Reply:
yeah, as long it's not in the stuff ran locally that's fine :)